map/models.py

```diff
@@ -105,6 +109,15 @@ def clean(self):
             if not lat and not lon:
                 raise ValidationError(_("This image has no GPS details" ))
 
+    def save(self, *args, **kwargs):
+        image_pil = Image.open(self.image)
+        new_image = resizeimage.resize_cover(image_pil, [300, 200])
```

Comment:
```suggestion
        image_pil.thumbnail([300, 200])
```
and drop the new requirement for `python-resize-image`